<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="CTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="navLayout">
          <item>
           <widget class="QLabel" name="label_Sort">
            <property name="text">
             <string>排序方法</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbox_Sort">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Sort_info">
            <property name="text">
             <string>排序指标</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbox_Sort_info">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Find_method">
            <property name="text">
             <string>查找方法</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbox_Find">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Find_info">
            <property name="text">
             <string>查找字段</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbox_Find_txt">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Txt_find"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>当前：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="currentPageLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>总页数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="countPageLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="upPushButton">
            <property name="text">
             <string>上一页</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="downPushButton">
            <property name="text">
             <string>下一页</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string>1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>新建列</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
           <item>
            <property name="text">
             <string>新建子项目</string>
            </property>
            <item>
             <property name="text">
              <string>新建子项目</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>新建项目</string>
             </property>
             <item>
              <property name="text">
               <string>新建子项目</string>
              </property>
             </item>
            </item>
           </item>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Tab3">
       <attribute name="title">
        <string>Tab 3</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QFontComboBox" name="fontComboBox"/>
        </item>
        <item>
         <widget class="QDial" name="dial"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CTabWidget</class>
   <extends>QTabWidget</extends>
   <header location="global">ctabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
